### ISSUE TITLE
Error initiating bootstrap connection to <ip> No route to host

### ISSUE THREAD

**Issue by Daan4** [nanocurrency/nano-node#3496]
### Summary

Starting bootstrapping gives this error every few seconds for a couple of IPs, most of which I could find with traceroute.

### Node version

V23.0DB2

### Build details

docker pull nanocurrency/nano-beta:V23.0DB2

### OS and version

Ubuntu 20.04.3 LTS

### Steps to reproduce the behavior

1. Obtain fresh Ubuntu install (strato vps host in my case)
2. Install docker
3. Run nano beta docker image with default settings

### Expected behavior

No error messages related to bootstrapping

### Actual behavior

See attached logfile

### Supporting files

[log.log](https://github.com/nanocurrency/nano-node/files/7305537/log.log)

**Comment by dsiganos** [comment]
All of these come from a non-standard port and therefore are likely to be peers behind a NAT, with translated port numbers, which do not accept incoming connections. I see nothing suspicious here. We should maybe be scoring the peers and backing off from trying to establish contact with problematic peers but that's simply an improvement.

**Comment by zhyatt** [comment]
Is this likely related to https://github.com/nanocurrency/nano-node/issues/3349?

**Comment by dsiganos** [comment]
Yes, it is related. But #3349 isn't a great solution either. #3349 will introduce a new set problems.
